about.html

```diff
@@ -0,0 +1,281 @@
+                                 </div>
+                             </div>
+                             <div class="card-content">
+                                 <h3 class="name">Mike Tyson</h3>
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+
+                 <div class="slide-content">
+                    <div class="card-wrapper">
+                         <div class="cardd">
+                             <div class="image-content">
+                                 <span class="overlay"></span>
+                                 
+                                 <div class="card-image">
+                                     <img src="./assets/images/cvimage2.JPG" alt="" class="card-img">
+                                 </div>
+                             </div>
+                             <div class="card-content">
+                                 <h3 class="name">Mike Tyson</h3>
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+            </article>
+            
+        </article>
+    </section>
+
+    <section class="footer-container">
+        <footer class="footer">
+            <article class="footer-top-container">
+                <!-- <article class="container footer-top">
+                    <h2>Our Partners</h2>
+                    <ul class="partner-logo">
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
```

Comment:
- Please note that to ensure your HTML is readable and easy to maintain, it is not advisable to leave commented-out code on your page. Please ensure that only the needed code or HTML template is present on your page 👍